src/microtime.cc

```diff
@@ -39,57 +39,57 @@ int gettimeofday(struct timeval *tv, struct timezone *tz) {
 #include <sys/time.h>
 #endif
 
-void Now(const Nan::FunctionCallbackInfo<v8::Value> &info) {
+Napi::Value Now(const Napi::CallbackInfo &info) {
   timeval t;
   int r = gettimeofday(&t, NULL);
 
   if (r < 0) {
-    return Nan::ThrowError(Nan::ErrnoException(errno, "gettimeofday"));
+    throw Napi::Error::New(info.Env(), "gettimeofday");
```

Comment:
The old code here retained the `errno` error code from C land in the error message, does the new Napi::Error does this automatically or is there a way we can preserve that behavior from nan?

Reply:
Hi @wadey,
you are right so I just updated the interested code. Now I create the error and than I set the code field like reported here:

```cpp
Napi::Error e = Napi::Error::New(info.Env(), "gettimeofday");
e.Set("code", Napi::Number::New(info.Env(), errno));
throw e;
```